data/Hrishi5111998.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Hrishikesh Tule",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Web developer **Digital Nomad**. open source enthusiast",
+  "avatar": "https://github.com/Hrishi5111998.png",
+  "links":[
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/@hrishikesh7030",
+      "icon": "FaTwitter"
+    },
+    {
+     "name": "Instagram",
+     "url": "https://instagram.com/@hrishikesh2412",
```

Comment:
Please fix your Instagram URL. It should be something like this! 👇 🛠

```suggestion
     "url": "https://instagram.com/hrishikesh2412",
```

![image](https://user-images.githubusercontent.com/88786642/212483278-dd3f0b9a-d2b3-424b-850e-53f99a40d94b.png)